benchmarks/pywrapfn.jl

```diff
@@ -10,7 +10,7 @@ end
 function PyWrapFn(o::Union{PyObject, PyPtr}, nargs::Int, returntype::Type=PyObject)
     pyargsptr = ccall((@pysym :PyTuple_New), PyPtr, (Int,), nargs)
     ret = PyNULL()
-    optr = o isa PyPtr ? o : o.o
+    optr = o isa PyPtr ? o : PyPtr(o)
```

Comment:
You can just do `optr = PyPtr(o)`, since this is the identity if `o isa PyPtr`.

Reply:
Or condense it to `return PyWrapFn{nargs, returntype}(pyincref_(PyPtr(o)), pyargsptr, PyNULL())`

Reply:
However, it looks like this benchmark will no longer work after 8d300021792b8146483e0df42afd11ccd665fd99, since I elminated that `_pycall!` method.

Reply:
Done.
What should I do about your last comment in this PR?